### ISSUE TITLE
Support 'exclude' for '-plugindependency'

### ISSUE THREAD

**Issue by jwausle** [ToToTec/cmvn#7]
Support 

```
        -plugindependency=GAV;\
         exclude=GA; \
         exclude=GA2; \
         exclude=GA3; \
```

with GAV=group:artifact:version
and GA  =group:artifact

**Comment by jwausle** [comment]
The last one '1c6c6242ae14c0c62344486767f34e59b9fd3d7e' is the right. The previous ones had java-format troubles. I reseted the commits. 

I´m wondering why Github know about this:|

**Comment by lefou** [comment]
I merged PR #8.
TODO: test case

**Comment by lefou** [comment]
Done
